### PR TITLE
`Development`: Fix lighthouse issues in logo

### PR DESCRIPTION
**Pull request by simon-hng**
#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.

### Motivation and Context

Fixes two lighthouse issues, one related to [accessibility](https://dequeuniversity.com/rules/axe/4.4/image-alt?utm_source=lighthouse&utm_medium=lr) and one regarding [performance](https://web.dev/optimize-cls/?utm_source=lighthouse&utm_medium=lr#images-without-dimensions)

### Description

Adds three attribute tags to the logo

### Steps for Testing
Open dev tools and look at the logo attributes

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Screenshots

image
image

### CODE REVIEW

**Review by LiaLism — DISMISSED** [review]
Three attributes are shown. Tested on ts1.

**Review by simon-hng — COMMENTED** [review]
change alt tag to empty string as the logo is purely decorative

**Review by tobias-lippert — DISMISSED** [review]
Tested on ts1. Attributes are visible in the dev console.
Code looks good as well.

File: src/main/webapp/app/shared/layouts/navbar/navbar.component.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h6 class="dropdown-header fw-medium" jhiTranslate="global.menu.guidedTutorial">
 <nav class="navbar navbar-dark jh-navbar" [class.expanded]="!isCollapsed">
     <div class="jh-logo-container">
         <a class="navbar-brand" routerLink="/" (click)="collapseNavbar()">
-            <img [src]="SERVER_API_URL + 'public/images/logo.png'" />
+            <img [src]="SERVER_API_URL + 'public/images/logo.png'" height="30" alt="" />
```
Comment (simon-hng) [inline src/main/webapp/app/shared/layouts/navbar/navbar.component.html:77]: alt is empty string, as the logo is only decorative. No width set in order to allow for easier change of logos

**Review by Strohgelaender — APPROVED** [review]

**Pull request merged**